contents/data/sql/SQL.md

```diff
@@ -0,0 +1,145 @@
+<frontmatter>
+  title: Introduction to SQL
+  footer: footer.md
+  head: head.md
+  siteNav: mainNav.md
+  pageNav: 3
+</frontmatter>
+
+{{ navbar | safe }}
+
+<div class="website-content">
+
+# Introduction to SQL
+
+**Author(s): [Amrut Prabhu](https://github.com/amrut-prabhu)**
+
+Reviewers: [Ronak Lakhotia](https://github.com/RonakLakhotia), [Rahul Rajesh](https://github.com/rrtheonlyone)
+
+## What is SQL?
+
+Databases are integral to any commercial software application, whether it is a web app, desktop app or otherwise.
+As the size of your application grows, you need to look into dedicated database software packages (like SQL software) to store your data. It will not be enough to simply rely on in-memory solutions or a simple text file to store the data. Many companies like Facebook, Google, Adobe, Paytm and Zappos use <tooltip content="Relational DataBase Management System">RDBMS</tooltip> software to fulfil this need.
+
+**Structured Query Language** (**SQL**) is a programming language that is specifically designed for managing (storing, retrieving and manipulating) data in a <tooltip content="relational databases store data in tables">[relational database]({{baseUrl}}/contents/data/databases/databases.html#database-models)</tooltip>. Unlike other languages, SQL doesn't come as a standalone installation. Rather, _SQL Software_ (or _SQL Implementations_) like MySQL, Oracle, SQL Server and PostgreSQL come with a RDBMS that uses the SQL language.
```

Comment:
Along the same vein, _MySQL, Oracle, SQL Server and PostgreSQL_ are not SQL implementations that comes with an RDBMS. They are RDBMS that come with an SQL implementation. Anyway, that's my understanding; but I'm not an expert.